.sync/azure_pipelines/MuDevOpsWrapper.yml

```
@@ -42,13 +42,10 @@ parameters:
   displayName: Perform Stuart PR Evaluation
   type: boolean
   default: true
-- name: calculate_code_coverage
-  displayName: Calculate Code Coverage From Unit Tests
-  default: false
 - name: coverage_publish_target
   displayName: Code Coverage Publish Target
   type: string

```

It seems a little more clear to add the allowed values under the `values` keyword.

```suggestion
- name: coverage_publish_target
  displayName: Code Coverage Publish Target
  type: string
  values:
  - 'ado'
  - 'codecov'
  - ''
```